woocommerce-services.php

```diff
@@ -664,7 +664,10 @@ public function attach_hooks() {
 			}
 
 			add_action( 'rest_api_init', array( $this, 'rest_api_init' ) );
-			add_action( 'woocommerce_settings_saved', array( $schemas_store, 'fetch_service_schemas_from_connect_server' ) );
+			add_action( 'update_option_woocommerce_store_postcode',  array( $schemas_store, 'fetch_service_schemas_from_connect_server' ) );
```

Comment:
Any way to ensure this is kept in sync with what the server consumes? I'm not sure how but maybe at least a code comment?

Reply:
> Any way to ensure this is kept in sync with what the server consumes?

I'm not sure what you mean by this.

Reply:
Referring to hypothetical future changes: if we were to update the server to act on more (or fewer) of the settings, we would likely want to modify this set of actions as well.

Reply:
Gotcha, I'll add a comment explaining this.

Reply:
Done in 32c198b. If we start making more changes on the server based on option values, we could look at adding those option keys to the schema and dynamically updating them in the future.